cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -81,8 +81,13 @@ Feature: Users can authneticate with OIDC authenticator
   Scenario: Non-existing username in ID token is denied
     Given I get authorization code for username "not_in_conjur" and password "not_in_conjur"
     And I fetch an ID Token
+    And I save the amount of log lines into "bookmark_not_in_conjur"
     When I authenticate via OIDC with id token
     Then it is unauthorized
+    And The log filtered from line number in "bookmark_not_in_conjur" should contains "1" messages:
```

Comment:
same here with the ammount of messages. if we use "1" all the time let's add a step named "The log filtered from line number in bookmark contains  message:"

and this will call the other step with the default bookmark and count = 1

Reply:
Thanks, i refactored please review